dataactvalidator/README.md

```diff
@@ -35,7 +35,7 @@ The validation process begins with a call to one of the above routes from either
 
 The file location on S3 is specified in the job tracker, and the validator streams the file record by record from S3.  The first row is the headers, which are checked against the file specification to ensure that all headers in the file are known fields and that no required fields are missing.
 
-Each record is checked against the set of rules for the fields present in the file.  If a record passes all rules, it goes into the staging database.  Otherwise, each failed rule goes into the error report, and a running sum is kept of error occurrences for each rule, which are written to the error database after all rows have been checked.  Finally, the job is marked as finished in the job tracker, the file is marked completed in the error database.
+Each record is checked against the set of rules for the fields present in the file.  With the exception of records that fail a data type check, each record goes into a staging table in the validation database. Data that fails any validation rule is written to the error report, and a running sum is kept of error occurrences for each rule, which are written to the error database after all rows have been checked.  Finally, the job is marked as finished in the job tracker, the file is marked completed in the error database.
 
```

Comment:
Thanks for cleaning up the documentation for this, it's good to keep that up to date.